aviatrix/resource_aviatrix_transit_external_device_conn.go

```diff
@@ -564,6 +571,16 @@ func resourceAviatrixTransitExternalDeviceConnCreate(d *schema.ResourceData, met
 		}
 	}
 
+	if val, ok := d.GetOk("enable_bgp_lan_activemesh"); ok && val.(bool) {
```

Comment:
`enable_bgp_lan_active_mesh` is TypeBool with default false so we shouldn't need to check with GetOK. We can just check Get: `d.Get("enable_bgp_lan_activemesh").(bool)`